### PR TITLE
added support for github org data provider

### LINKED ISSUE #115: Enable github org processor
### What do you want to improve?

Enable the github org processor, to automatically import developers and groups from github org members and github org teams.
 Please refer to this document: https://backstage.io/docs/integrations/github/org

### :warning: Is this a breaking change?

_Put an `x` in the boxes that apply_

- [ ] Yes
- [ ] No
- [x] Don't know

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
## Description

added support for github org data provider.

The backstage example had some information hardcoded. @christophe-f can you help me parameterize them?

## Which issue(s) does this PR fix

- Fixes #115 

## PR acceptance criteria

_Put an `x` in the boxes that apply_

- [ ] GitHub Actions are completed and successful
- [ ] Unit Tests are updated and passing
- [ ] E2E Tests are updated and passing
- [ ] Documentation is updated if necessary

### CODE REVIEW

File: packages/backend/src/plugins/catalog.ts [hunk 1]
```diff
@@ -26,6 +27,18 @@ export default async function createPlugin(
...
   );
 
+  builder.addEntityProvider(
+    GithubOrgEntityProvider.fromConfig(env.config, {
+      id: 'production',
+      orgUrl: 'https://github.com/raf-backstage-demo',
+      logger: env.logger,
+      schedule: env.scheduler.createScheduledTaskRunner({
+        frequency: { minutes: 60 },
+        timeout: { minutes: 15 },
```
Comment (riginoommen) [inline packages/backend/src/plugins/catalog.ts:37]: Including the  initalDelay parameter for syncing helps

Comment (raffaelespazzoli) [reply packages/backend/src/plugins/catalog.ts:37]: is there any change you need here?

Comment (raffaelespazzoli) [reply packages/backend/src/plugins/catalog.ts:37]: @riginoommen

Comment (raffaelespazzoli) [reply packages/backend/src/plugins/catalog.ts:37]: @riginoommen how would it help? I mean, why would waiting be a good thing? Also what value do you want for the initialDelay?

Comment (raffaelespazzoli) [reply packages/backend/src/plugins/catalog.ts:37]: done

**Review by riginoommen — APPROVED** [review]
LGTM

**Review by christophe-f — CHANGES_REQUESTED** [review]
Need to externalize the GitHub organization. Cannot be hardcoded.

**Comment by raffaelespazzoli** [comment]
@christophe-f I agree, can you show m,e how to do it? I don't know how to read from properties in typescript and in particular in backstage.

**Comment by christophe-f** [comment]
Once the conflicts are resolved, it's good to merge.

**Review by christophe-f — APPROVED** [review]
LGTM

**Pull request merged**